app/utils/draft.js

```diff
@@ -79,6 +83,70 @@ export function alertSendToGroups(formatMessage, notifyAllMessage, accept, cance
     );
 }
 
+export function tryOpenURL(url) {
+    Linking.canOpenURL(url).then((supported) => {
```

Comment:
I believe this caused issues in the past with iOS 14, @migbot can you confirm?

Reply:
Is there a reason for reverting? If you think there is, go ahead

Reply:
Just to make sure. Can I use "canOpenURL" here, or should I change anything?

Reply:
You can leave it as is